main.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable linebreak-style */
+
+const list = document.getElementById('list');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const buttonClass = document.querySelector('.buttonClass');
+const books = [];
+
+function saveBooks(newBooks) {
+  // books = newBooks;
```

Comment:
- [ ] Please remove this commented code line as it makes the codebase untidy and you can always refer to GitHistory if you need to refer to previous code versions.